test/functional/rpc_blockchain.py

```diff
@@ -125,10 +125,10 @@ def _test_getblockchaininfo(self):
         assert_greater_than(res['size_on_disk'], 0)
 
         assert_equal(res['softforks'], {
-            'bip34': {'type': 'buried', 'active': False, 'height': 500},
+            'bip34': {'type': 'buried', 'active': False, 'height': 1000},
             'bip66': {'type': 'buried', 'active': False, 'height': 1251},
             'bip65': {'type': 'buried', 'active': False, 'height': 1351},
-            'csv': {'type': 'buried', 'active': False, 'height': 432},
+            'csv': {'type': 'buried', 'active': False, 'height': 720},
```

Comment:
this is incorrect I think?

Reply:
Matches the `720` in `chainparams.cpp` line 392